CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.5.1 (2019-03-19)
+
+- Fixed `SkyI18nModule` to not have a circular reference to `@skyux/core`. [#23](https://github.com/blackbaud/skyux-i18n/pull/23)
```

Comment:
What about "Fixed `SkyI18nModule` to remove a circular reference to `@skyux/core`." ? Would saying "remove" change the meaning?

Reply:
Works for me. Done.